lib/pick.js

```diff
@@ -2,7 +2,30 @@
 
 module.exports = function(obj, keys) {
   return keys.reduce((accum, key) => {
-    accum[key] = obj[key];
+    if (!key.includes('.')) {
+      // Regular keys, just add them
+      accum[key] = obj[key];
+      return accum;
+    }
+
+    // Complex keys, let's split them out
+    const [parent, child] = key.split('.');
```

Comment:
What if I want `attrs.negative.runtime`?

Reply:
I did this intentionally. The idea is to keep it simple and avoid much complexity. My use case is only for fetching some of the attrs properties to avoid a large dump of unnecessary info. If I want to fetch the negative property, I wouldn't worry too much of getting more properties there.

My vote is to keep it this way, not going too deep for now. If we have a use case where we need more layers of properties, I'd be happy to update this code.

What do you think?

Reply:
Ok, that's fine then.